project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala

```diff
@@ -138,20 +132,22 @@ case class MacDistHandler(jre: Option[String], jreName: String)
         new File(appDir.getPath).toPath,
         new SimpleFileVisitor[Path]() {
           override def visitFile(
-              file: Path,
-              attrs: BasicFileAttributes
+            file:  Path,
+            attrs: BasicFileAttributes
           ): FileVisitResult = {
+            if (file.toFile.getName.startsWith("scala-compiler")) {
+              println(s"Remove native libs from scala-compiler jar")
+              val cmd = Seq("zip", "-d", file.toFile.getAbsolutePath, """META-INF/native/**""" )
+              cmd.lines.foreach(println)
```

Comment:
Where does it actually do the removal?

Reply:
When you call `cmd.lines` it  executes and produces an output

Reply:
Seems to be a method on `ProcessBuilder`, I wonder how `Seq` becomes a `ProcessBuilder`.  But I don't wonder enough to investigate. :+1:

Reply:
It is syntax added on the scala lib
https://www.scala-lang.org/api/2.12.8/scala/sys/process/index.html